researchstudio-sat/webofneeds
# Feat  change notification type

Changes need modification notification as its own messageType and not as a connectionMessage with a certain text -> and process this new message within the won-owner-webapp:

How to test:
- try asking for a need modification within a connection to the debug bot
- see that nothing is different and everything works just the way it did before
